ui/react-components/components/Conflicts/Conflicts.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ConflictsFooter from "./Footer/ConflictsFooter.jsx";
+import ConflictsBody from "./Body/ConflictsBody.jsx";
+import PropTypes from "prop-types";
+import {injectIntl} from "react-intl";
+
+const Conflicts = props => {
+
+    return (
+        <div className="conflicts">
+            <div className={"body"}>
+                <ConflictsBody conflicts={props.conflicts} service={props.service}/>
```

Comment:
Instead of Conflicts body can we have Different Component for Service and other conflicts?

Reply:
@bsneha90 extracted, please review